src/main/java/org/sonarlint/intellij/finding/LiveFinding.java

```diff
@@ -64,6 +64,7 @@ public LiveFinding(Issue issue, PsiFile psiFile, @Nullable RangeMarker range, @N
     this.uid = UID_GEN.getAndIncrement();
     this.context = context;
     this.quickFixes = quickFixes;
+    this.creationDate = System.currentTimeMillis();
```

Comment:
For an issue that is detected on the first analysis of a file, we keep the creation date to null. Is it still supported?

Reply:
No. Why do we need to keep it null ? The only difference today is when there is no creation date on an issue, we don't display "a few seconds ago" after the issue message. If we need to know that the issue is new, I suggest we use another field